server/services/prisonService.test.ts

```diff
@@ -208,4 +214,36 @@ describe('prisonService', () => {
       expect(prisonRegisterStore.setActivePrisons).toHaveBeenCalledWith(allPrisons, 1)
     })
   })
+
+  describe('lookupPrison', () => {
+    it('should lookup prison by ID', async () => {
```

Comment:
Thank you for having a stab at writing the tests, but theres a couple of things not quite right here

Firstly, what you've written is a test of a mock - you have ended up testing a mock! 🤣 
On line 224 you setup the `prisonService.lookupPrison` as a mock, but then on line 227 you invoke that same mock!

Line 227 is correct - we want to test the new method you've added to prisonService, ie. lookupPrison

But the thing you need to mock is not the method you are testing, but instead the internal dependency of that method. In this case it would probably be the prisonRegisterStore (I'd have to check which methods call what etc, but I think thats right). 

The 2nd thing thats not right is your second test. What we want to be testing here is the catch block of the `prisonService.lookupPrison` method. In order to do that we need to setup the mock (probably prisonRegisterStore - see above) to throw an exception. By doing that we then force prisonService.lookupPrison to go into it's catch block 👍 

I'll checkout your branch in a mo and see if I can help with it